subworkflows/local/profiling.nf

```diff
@@ -264,7 +264,7 @@ workflow PROFILING {
                                                 .filter { meta, db -> meta.tool == 'centrifuge' }
                                                 .map { meta, db -> [meta.db_name, meta, db] }
 
-        ch_input_for_centrifuge_kreport = combineProfilesWithDatabase(CENTRIFUGE_CENTRIFUGE.out.report, ch_database_for_centrifugekreport)
+        ch_input_for_centrifuge_kreport = combineProfilesWithDatabase(CENTRIFUGE_CENTRIFUGE.out.results, ch_database_for_centrifugekreport)
```

Comment:
We are not breaking something here again (as in reverting again?)

Reply:
I think because I didn't update 'dev' when creating 'review' branch. I think it should be "results"